src/index.ts

```diff
@@ -248,6 +258,9 @@ export class IdsToolbox {
       if (!(await this.preflightRequireNix())) {
         this.recordEvent("preflight-require-nix-denied");
         return;
+      } else {
+        await this.preflightNixStoreInfo();
+        this.addFact(FACT_NIX_STORE_TRUST, this.nixStoreTrust);
```

Comment:
Should we exit with an error / log a warning if they're not `trusted`?

Reply:
Most of our actions don't need trust, so I'm thinking we'll leave it up to those actions: https://github.com/DeterminateSystems/magic-nix-cache-action/pull/52